### DISCUSSION TITLE
Number of solutions

### DISCUSSION THREAD

**Discussion by ErikLambrechts** [coin-or/python-mip#257] [Q&A] [answered]
I am working on a problem where I have a feasible solution to a set of constraints. I don't have an objective but I trying to figure out if there are other feasible solutions satisfying this set of constraints. 

Is there an efficient why to solution this?

**Comment by sebheger** [comment, marked as answer]
@ErikLambrechts. Best to my knowledge python-mip doesn't provide you with an efficient way. You can try two approaches (without guarantee):

1.  Perform runs with different artificially generated target functions to see whether you can find a different solution. This approach might "fail" if there is only one feasible solution or you do not find the required target function to produce different output.
2. If we are talking to integer programming, you can try to add an additional constraint that exactly forbids your feasible (integral) solution. If the new model has no integral solution, you know that there is only one. Otherwise, you get the other one. Depending on your problem if/how this constraint can/must be constructed.
